View/style.css

```diff
@@ -0,0 +1,137 @@
+* {
+  margin: 0px;
+  padding: 0px;
+  display: block;
+  box-sizing: border-box;
+}
+
+img {
+  width: 100%;
+  height: 100%;
+}
+
+#game {
+  position: absolute;
+  height: 800px;
+  width: 800px;
+  background-color: bisque;
+  overflow: hidden;
+}
+
+#space {
+  width: 100%;
+  height: 50%;
+  position: relative;
+}
+
+.trash {
+  display: inline-block;
+  width: 10%;
+  height: 10%;
+  background-color: black;
+  z-index: 1;
+  position: absolute;
+}
+
+.banana {
```

Comment:
This and all the rest are the same as the .trash. Make some general class, and give it the general options, after that, add only the specific details such as background and the like